app/views/my_projects/index.html.erb

```diff
@@ -0,0 +1,16 @@
+<h1><%= t('my_projects.header') %></h1>
+
+<h2><%= t('my_projects.approved_projects_header') %></h2>
+<div data-type="approved-projects">
```

Comment:
Uncomfortable with making a div here just for test grouping purposes -- excess divs can impede our ability to CSS nicely. You seem uncomfortable with the idea too -- do we have another way of expressing this?

Non-blocking.

Reply:
We could definitely add a `data-status` attribute which will let us find all the projects with status pending/approved/rejected that way it's not tied to the page layout/grouping